tile/lib/lib.cc

```diff
@@ -279,80 +276,90 @@ lang::RunInfo LoadConstCalc(const std::string& name) {
   DoubleN(i) = N();
   Tensor Partial = Simple + DoubleN;
   Tensor O = Partial + F2;
-  return Evaluate(name, {O});
+  return Program(name, {O});
 }
 
-lang::RunInfo LoadConv1d(const std::string& name,     //
-                         const LogicalShape& input,   //
-                         const LogicalShape& kernel,  //
-                         const std::vector<int64_t>& output) {
+Program LoadConv1d(              //
+    const std::string& name,     //
+    const LogicalShape& input,   //
+    const LogicalShape& kernel,  //
+    const std::vector<int64_t>& output) {
   auto I = Placeholder(input, "I");
   auto K = Placeholder(kernel, "K");
-  auto runinfo = Evaluate(name, {Convolution(I, K, output)});
-  runinfo.const_inputs = {"K"};
-  runinfo.input_buffers = {{"K", MakeBuffer(kernel)}};
-  return runinfo;
+  return Program(name, {Convolution(I, K, output)});
+  // auto runinfo = Evaluate(name, {Convolution(I, K, output)});
+  // runinfo.const_inputs = {"K"};
+  // runinfo.input_buffers = {{"K", MakeBuffer(kernel)}};
+  // return runinfo;
```

Comment:
Are the various runinfo comments in this file reminders to add `const_inputs` and `input_buffers` functionality/testing?

Reply:
I will remove these comments. The` runinfo` was mixing concerns; I have a new `Binder` object on the Execution API that should provide similar functionality. Marking constant buffers is still an open item however.